docs/topics/js/js-overview.md

```diff
@@ -59,6 +59,61 @@ at the expense of type safety.
+
+_KVision_ is an object-oriented web framework that makes it possible to write applications in Kotlin/JS without having to
+write HTML or CSS. Instead, it provides ready-to-use components that can be used as building blocks for your application’s
+user interface. You can use both reactive and imperative programming models to build your frontend, use connectors for Ktor,
+Spring Boot, and other frameworks to integrate it with your server-side applications, and share code using Kotlin Multiplatform.
+
+Visit [https://kvision.io](https://kvision.io) for documentation, tutorials, and examples.
+
+For updates and discussions about the framework, join [#kvision](https://kotlinlang.slack.com/messages/kvision) and
+[#javascript](https://kotlinlang.slack.com/archives/C0B8L3U69) channels in the [Kotlin Slack](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up).
+
+### fritz2
+
+_fritz2_ is a standalone framework for building reactive web user interfaces. It provides its own type-safe DSL for building
+and rendering HTML elements, and it makes use of Kotlin’s coroutines and flows to express components and their data bindings.
+It provides state management, validation, routing, and more out of the box, and integrates with Kotlin Multiplatform projects.
+
+Visit [https://www.fritz2.dev](https://www.fritz2.dev) for documentation, tutorials, and examples.
+
+Join the [#fritz2](https://kotlinlang.slack.com/messages/fritz2) and [#javascript](https://kotlinlang.slack.com/archives/C0B8L3U69)
+channels in the [Kotlin Slack](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up)to stay up to date with the latest
+news about this framework.
+
+### Doodle
+
+_Doodle_ is a vector-based UI framework for Kotlin/JS. Apps built using it do not rely on DOM, CSS, or Javascript. Instead,
+they use the browser’s graphics capabilities to draw user interfaces, giving you precise control over the rendering of
+arbitrary UI elements, vector shapes, gradients, and custom visualizations.
+
+Visit [https://nacular.github.io/doodle/](https://nacular.github.io/doodle/) for documentation, tutorials, and examples.
+
+Join [#doodle](https://kotlinlang.slack.com/messages/doodle) and [#javascript](https://kotlinlang.slack.com/archives/C0B8L3U69)
+channels in the [Kotlin Slack](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up) for updates and discussions about
+this framework.
+
+### Compose for Web
+
+_Compose for Web_ brings [Google’s Jetpack Compose UI toolkit](https://developer.android.com/jetpack/compose) to your browser.
+It allows you to build reactive web user interfaces using the concepts introduced by Jetpack Compose. It provides a DOM API
+to describe your website, as well as an experimental set of multiplatform layout primitives. Compose for Web also gives
+you the option to share parts of your UI code and logic across Android, desktop, and web.
+
+Compose for Web is in _Technology Preview_, which means it is ready for experiments but not yet ready for production use.
```

Comment:
This information is extremely difficult to keep up to date.

Maybe give a link with the text "pay attention to the status of the feature" or smth like that?

https://developer.android.com/jetpack/androidx/versions/all-channel

Reply:
Indeed it is. But I think it's crucial to have a note about unstable/non-production stage right here because we list Compose among other frameworks that don't have such concerns.

When Compose for Web graduates, we'll have to adjust our docs anyway (add examples, provide link to complete docs, whatever), so I don't think this point is worth removing important info about the current state.

Reply:
OK, sounds reasonable. So let's don't miss the stable version of Compose :)